packages/salesforcedx-vscode-core/package.nls.json

```diff
@@ -70,6 +70,7 @@
   "force_analytics_template_create_text": "SFDX: Create Sample Analytics Template",
   "force_function_create": "SFDX: Create Function",
   "force_function_start": "SFDX: Start Function",
+  "force_function_start_container": "SFDX: Start Container Function",
```

Comment:
New command that will run the old container based start function code. @sbudhirajadoc